app/bundles/LeadBundle/EventListener/OwnerSubscriber.php

```diff
@@ -98,14 +98,17 @@ private function getGeneratedTokens(EmailSendEvent $event)
         if (!$owner) {
             return $this->getEmptyTokens();
         }
+        $tokens          = [];
+        $combinedContent = $event->getCombinedContent();
+        foreach (self::onwerColumns as $ownerColumn) {
+            $token = $this->buildToken($ownerColumn);
+            if (false !== strpos($combinedContent, $token)) {
+                $ownerColumnNormalized = str_replace(['firstname', 'lastname'], ['first_name', 'last_name'], $ownerColumn);
+                $tokens[$token]        = ArrayHelper::getValue($ownerColumnNormalized, $owner);
```

Comment:
Please use PHP 8 syntax rather than ArrayHelper. I'd like to deprecate it now when PHP offers nice syntax to writ the same.
```suggestion
                $tokens[$token]        = $ownerColumnNormalized[$owner] ?? null;
```